osu.Game/Rulesets/Mods/ModTimeRamp.cs

```diff
@@ -86,6 +86,23 @@ public double ApplyToRate(double time, double rate = 1)
             return rate * Math.Round(ramp, 2);
         }
 
+        public double GetTimeAt(double time)
```

Comment:
should this be an abstract method in `IApplicableToRate`, considering how `bakeClockTime` is written? something along those lines

Reply:
Correct, it would be better suited if `IApplicableToRate` mods could do this conversion themselves.